.github/actions/spell-check/expect.txt

```diff
@@ -1943,6 +1943,7 @@ stdout
 stefan
 STEPIT
 stgm
+STGM
```

Comment:
I think the system is confused because of this.

@jsoref
What do you think is the reason why `STGM` still gets tagged as incorrect?